tests/pr_test.go

```diff
@@ -54,6 +55,35 @@ func TestRunBasicExample(t *testing.T) {
 	assert.NotNil(t, output, "Expected some output")
 }
 
+func TestRunAdvanceExample(t *testing.T) {
+	t.Parallel()
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Prefix:  "advanced-key-protect",
+		TarIncludePatterns: []string{
+			"*.tf",
+			"modules/*/*.tf",
+		},
+
+		ResourceGroup:          resourceGroup,
+		TemplateFolder:         advancedExampleTerraformDir,
+		Tags:                   []string{"test-schematic"},
+		DeleteWorkspaceOnFail:  false,
+		WaitJobCompleteMinutes: 60,
+	})
+
```

Comment:
@ocofaigh after adding this line, i started to get the below error:

pr_test.go:1: : # github.com/terraform-ibm-modules/terraform-ibm-key-protect [github.com/terraform-ibm-modules/terraform-ibm-key-protect.test]
./pr_test.go:77:76: undefined: region (typecheck)

Before this the CI ran successful

Reply:
@ocofaigh can you please try running the pipeline now.

Reply:
@marifse thats because you don't have any region variable defined. Try this:
```
// Workaround for https://github.com/IBM-Cloud/terraform-provider-ibm/issues/5154 options.AddWorkspaceEnvVar("IBMCLOUD_KP_API_ENDPOINT", "https://private."+options.Region+".kms.cloud.ibm.com", false, false)
```

Reply:
@ocofaigh done. I also tried that but that time it did not work at first. Anyways now it is okay.